### PR TITLE
Sils/simply

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Comment by Nosferatul** [comment]
ack 8890d0e

**Comment by sils** [comment]
reack https://github.com/coala/coala/pull/3527/commits/8890d0e0db64571fe569e6ca9664b02f8949c8be

**Comment by sims1253** [comment]
ack 1b976e5cf9aa13c03ffdf082abb8fa0643e2e7a3

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -201,23 +214,18 @@ def print_lines(console_printer,
 
         printed_chars = 0
         if i == sourcerange.start.line and sourcerange.start.column:
-            console_printer.print(highlight_text(
-                no_color, line[:sourcerange.start.column - 1], lexer), end='')
+            print_normal(line[:sourcerange.start.column - 1])
```
Comment (sims1253) [inline coalib/output/ConsoleInteraction.py]: BackgroundSourceRangeStyle was not in the old call but in the new one.
So maybe this also is a print_normal?

Comment (sils) [inline coalib/output/ConsoleInteraction.py]: yes it is, what's wrong?

**Comment by sils** [comment]
we're getting a coverage issue on the CI because the code previously was only covered because one of the lines was doubly run... I'm going to remove that commit changing this and let the rest be like this

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (sims1253) [inline coalib/output/ConsoleInteraction.py]: must have skipped a line. Thought you used the other method... Fine then :)

**Pull request closed without merging**